Menu/Menu.less

```diff
@@ -6,7 +6,7 @@
 
 .menu {
   display: none;
-  position: fixed;
+  position: relative;
```

Comment:
this is causing a problem where the menu is rendering ABOVE the articles and pushing them out of sight when the menu opens.